packages/keystone/test.utils.js

```diff
@@ -710,6 +710,37 @@ const expectToThrowUniqueConstraintViolationError = async (testFunc, constraintN
     })
 }
 
+/**
+ * @param testFunc
+ * @param {string} path
+ * @param {string} field
+ * @param {Number} [count]
+ * @returns {Promise<void>}
+ */
+const expectToThrowAccessDeniedToFieldError = async (testFunc, path, field, count = 1) => {
```

Comment:
basic `expect` + `path` won't work here?

Reply:
I'd like to test the access to a particular field, so it's necessary to add the field name. I have had to add the field as a separate parameter due to the structure within the caught error object.
Need to check `caught -> errors -> n -> path`. The `path` is an array `[path, n, field]`.
At the same time need to check `caught -> data`. The `data` is an object `{path => array}`.
<img width="847" alt="image" src="https://github.com/open-condo-software/condo/assets/25384290/71e805e2-2be2-4cac-acd0-630ddff32f7b">
The current decision is fit to solve the problem, but of course, it may be changed in the future.